docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Common Installation Errors
+
+This guide assumes installation on any supported OS, and was primarily written based on an Ubuntu 18 stack.  
+
+This guide offers support for common issues faced when first installing LORIS. It is assumed that all installation steps provided have been 
+correctly followed and that all required dependencies have been installed.
+
+## MySQL 8 and PHP 7.3
+
+If you are running into issues when setting up your MySQL or MySQL user account and you are using MySQL 8 and PHP 7.3 
+(instead of PHP 7.4), please read [this link](https://www.php.net/manual/en/mysqli.requirements.php) and also [this](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password-compatible-connectors)
+
+## Frontend Does Not Load/JavaScript changes not visible
+The first command to try to run (under the _loris root_ directory):
+
+```bash
+make clean
+```
+
+Next, run one of the following commands depending on your environment:
+
+```bash
+make        # For production environments
+make dev    # For development environments
+```
+
+If that does not solve your issue, this could be because the Config table in the database is not pointing to the correct values. In this case, follow the following steps. 
+
+1. Check the following values in the Config table via the MySQL backend.
+
+   | Name | Label          | Value          |  
+   | ---- | ----------- | ---------- |
+   | base | Base           | $lorisRootDirectory |
+   | host | Host           | $hostName  | 
+    
+   These values can be viewed by running the following commands:
+    
+   ```sql
+    SELECT c.configID, cs.Name, cs.Label, c.value, cs.Description FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='base');
+    SELECT c.configID, cs.Name, cs.Label, c.value, cs.Description FROM Config c LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+   ```
+
+2. If they do not match the above format, update these values with the following commands on the MySQL command-line.
```

Comment:
I don't understand what you mean by "match the above format"

Reply:
I meant to say if they don't match your $lorisRootDirectory and $hostName. I can change this wording to make it more clear!